docs/reference/ccr/apis/follow/post-pause-follow.asciidoc

```diff
@@ -48,6 +48,15 @@ POST /<follower_index>/_ccr/pause_follow
 `follower_index` (required)::
   (string) the name of the follower index
 
+
+==== Authorization
+
+If the {es} {security-features} are enabled, to use this API you must have 
```

Comment:
Sorry, I should have better described my comment in the other PR, but in the case of pause follow, only `manage_ccr` is needed. The reason is that it just stops the follow tasks. (resume follow is different as it start shard following and then uses the privileges of the user that invoked resume follow api to read from leader and write in into follower)